src/test/java/com/shopify/model/ShopifyCustomCollectionCreationRequestTest.java

```diff
@@ -0,0 +1,23 @@
+package com.shopify.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ShopifyCustomCollectionCreationRequestTest {
+
+	private static final String SOME_TITLE = "Basic collection";
+	private static final String SOME_HANDLE = "basic-collection";
+
+	@Test
+	public void simpleProductCreationRequest() {
+		final ShopifyCustomCollectionCreationRequest customCollectionCreationRequest = ShopifyCustomCollectionCreationRequest.newBuilder()
```

Comment:
Like you stated in the PR description, please add more test coverage on your ShopifySDK method.

Reply:
Working on it. New changes will be pushed once more tests are added